docs/intro/intro_symbolic_math.rst

```diff
@@ -0,0 +1,85 @@
+.. _intro_symbolic_math:
+
+=============
+Symbolic Math
+=============
+
+dimod enables easy incorporation of binary and integer variables as
+:ref:`single-variable models <symbolic_math_generators>`. For example, you can
+represent such binary variables as follows:
+
+>>> from dimod import Binary, Spin
+>>> x = Binary('x')
+>>> s = Spin('s')
+>>> x
```

Comment:
Overall this seems like a place that we should be using the `to_polystring()` representations rather than printing the models.

Reply:
This one is to show the generated BQM but definitely a better output for `x + s` and such that use the variables in some formula